src/apps/sources/pages/ViewSourcePage.tsx

```diff
@@ -103,7 +103,7 @@ export const ViewSourcePage: React.FC<Props> = ({
     retrieveSourceAndDetails(url);
   }, [url, retrieveSourceAndDetails]);
   useEffect(() => {
-    retrieveConceptsSummary({conceptsUrl: `${url}concepts/`, limit: 1, includeRetired: true});
+    retrieveConceptsSummary({conceptsUrl: `${url}concepts/`, limit: 1, includeRetired: false});
```

Comment:
For these purposes, we do actually want to include retired concepts, since that's part of what we display to users:

<img width="376" alt="Screenshot of the concepts from a source" src="https://user-images.githubusercontent.com/52504170/113161715-a62a8780-920c-11eb-8b04-e2d3447f9b5e.png">